infrastructure/general/.pages

```diff
@@ -2,5 +2,5 @@ title: General Infrastructure
 nav:
   - index.md
   - 'Setup Secure WebSocket': setup-secure-wss.md
-  - 'Set Up an RPC Node': 'setup-rpc.md'
-  - 'Set Up a Bootnode': 'setup-bootnode.md'
+  - 'Setup an RPC Node': 'setup-rpc.md'
+  - 'Setup a Bootnode': 'setup-bootnode.md'
```

Comment:
For titles of guides that have a set of instructions on how to achieve X, it's typically better to use the verb form of the title. This helps clarify what the user is expected to do or learn from the page.

So, in this case, "Set Up an RPC Node" is clearer for a page that guides users through the process of configuring an RPC node.

However, if the page is more of an overview or a guide that describes a concept, the noun form could be appropriate. For example, "RPC Node Setup" could be used for a page that gives a general explanation of what a node setup involves without focusing directly on performing the steps.